step2-01/README.md

```diff
@@ -2,54 +2,81 @@
 
 [Lessons](../) | [Exercise](./exercise/) | [Demo](./demo/)
 
-In this step, we'll cover enough of the TypeScript concepts to be productive with the React & Redux frameworks.
+In this step, we'll cover enough TypeScript concepts to be productive with the React & Redux frameworks.
 
 Topics in this step will include:
 
-- ES modules
-- Basic Types
-- Interfaces & Classes
-- Basic Generics
-- Spread and Destructuring
-- Async / Await
+- [ES modules](#modules)
+- [Types](#typescript-types)
+- [Spread](#spread-operator) and [Destructuring](#destructuring)
+- [Promise](#promise) and [Async / Await](#async--await)
+
+> To try out TypeScript concepts and see the corresponding JavaScript, you can use the [TypeScript playground](http://www.typescriptlang.org/play/index.html). We won't be using it in this training, but it's very handy in general!
 
 ## Modules
 
-Historically, JS is only executed in browser. The code all had to be loaded from `<script>` tags. Since the introduction of node.js, the JS community needed a way to scale beyond just single script files. Other language support the notion of modules. There are many JS modularity standards today.
+Historically, JS was only executed in-browser. The code all had to be loaded using `<script>` tags. With the introduction of node.js, the JS community needed a way to scale beyond just single script files. Other languages support the notion of modules, so various groups started developing modularity standards for JS.
 
 The most important ones to know about are:
 
-- commonjs - Node.js's standard to support modules
-  - synchronous
-  - require() function, can be dynamically called in the course of a program
-- ESM (ECMAScript module) - language level support
+- **commonjs** - Node.js's standard to support modules
+  - synchronous loading using `require()` function
+  - `require()` can be dynamically called in the course of a program
+- **ESM (ECMAScript module)** - language-level support
   - statically analyzable and synchronous
   - dynamic and asynchronous support via `import()` that returns a promise
 
+> For more information about the *many* modularity patterns and standards developed over time, see [this article](https://medium.freecodecamp.org/javascript-modules-a-beginner-s-guide-783f7d7a5fcc). You may still encounter some of the older patterns in legacy code.
+
 ## TypeScript Types
 
-Refer to the `demo/src` for some examples of some of the types available in TS that benefits a React developer.
+Refer to [`demo/src`](./demo/src) for examples of some of the types available in TS that benefit a React developer.
 
-## Spread Syntax
+## Spread Operator
```

Comment:
This is a bit redundant with material Micah included in day 1, since he's assuming an ES6-compatible browser. Could still be worth including though in case people skipped day 1.

Reply:
I'm probably gonna repeat because this stuff is the thing that bit a lot of devs when I was in OWA :P